geometry_msgs/Pose grasp_pose
geometry_msgs/Pose place_pose
float64[] object_vertices